KNOWN_ISSUES.md

```diff
@@ -72,3 +72,38 @@ consult the Cloudant documentation for further information.
   ```
   json: cannot unmarshal string into Go value of type map[string]json.RawMessage
   ```
+
+### Bodies containing the `headers` parameter
+
+The `Headers` map is reserved for the API request headers.
```

Comment:
```suggestion
The `Headers` field is always reserved for the request headers.
```

Reply:
I was brave enough to say it is a map, because the Go-wrapped headers type is a map. I also used `API request headers` because the comment above each `Headers` says the same. E.g:
https://github.com/IBM/cloudant-go-sdk/blob/fc0ed4d90c1ef0c8ee7f019455a3d47decbd7ad6/cloudantv1/cloudant_v1.go#L9110-L9111